benchmarks/benches/lib.rs

```diff
@@ -650,6 +651,30 @@ fn insert_range_bitmap(c: &mut Criterion) {
     }
 }
 
+fn insert_range_treemap(c: &mut Criterion) {
+    for &size in &[10, 100, 1_000, 5_000, 10_000, 20_000] {
+        let mut group = c.benchmark_group("insert_range_treemap");
+        group.throughput(criterion::Throughput::Elements(size as u64));
+        group.bench_function(format!("from_empty_{}", size), |b| {
+            let bm = RoaringTreemap::new();
+            b.iter_batched(
+                || bm.clone(),
+                |mut bm| black_box(bm.insert_range(0..size)),
+                criterion::BatchSize::SmallInput,
+            )
+        });
+        group.bench_function(format!("pre_populated_{}", size), |b| {
+            let mut bm = RoaringTreemap::new();
+            bm.insert_range(0..size);
```

Comment:
Also we always start at zero when we call `insert_range`, we should either use bigger number or make sure we are on the boundary of two bitmaps. Because, here, we are benchmarking only the `RoaringBitmap::insert_range`.

But not sure we can do much better!